index.bs

```diff
@@ -1962,7 +1963,7 @@ following Web IDL.
     This structure is used by the client to compute the following quantities:
 
     : <dfn dfn>JSON-serialized client data</dfn>
-    :: This is the [=UTF-8 encoding=] of the result of calling the initial value of {{JSON/stringify|JSON.stringify}} on a
+    :: This is the [=UTF-8 encoding=] of the result of calling [=%JSON%=].{{JSON/stringify|stringify}} on a
```

Comment:
WebIDL/browser-guts experts will need to review this. -- "the initial value of" is a magic incantation, i am not sure [=%JSON%=] is equivalent/better.

Reply:
I agree this needs expert review. As I understood @annevk's [comment](https://github.com/w3c/webauthn/issues/712#issuecomment-366196187) I think %JSON% should unambiguously identify "the initial value", which is why I dropped that, but I could very well be wrong.

Reply:
@domenic can probably review this. I suspect this is wrong given what we ended up with for the parse situation in Infra.

Reply:
This seems like a step backward because now, while you're using the initial value of the JSON object, you're no longer using the initial value for the stringify function. So the new version would be robust to `window.JSON = {}`, but would not be robust to `JSON.stringify = () => {};`.

I think what's there for now is fine-ish. A fully formalized version of this would require adding a %JSONStringify% intrinsic to the JS spec, then writing something like `! Call(%JSONStringify%, undefined, « a CollectedClientData dictionary, converted to a JavaScript object »)`.

(Side note: this spec seems to imply you're specifically calling with no third argument to JSON.stringify, so no spaces will be produced in the output. That's worth testing.)

Reply:
Ok. Would "the initial value of %JSON%.stringify" fix that?

I _think_ it ultimately shouldn't matter whether the client produces pretty or compact JSON (or whether different browsers decide differently on that), as the JSON result is returned verbatim to the RP for verification anyway.

Reply:
I think it's better to stick with the original text, as it's at least clear that it's not going for full-precision. "Initial value of %JSON%.stringify" is in an uncanny valley.

Reply:
Ok, in that case I suggest we close this and #712. Thanks for the reviews!

Reply:
agreed, thx for reviews :)